### PR TITLE
AF-1067: [Project Oriented] Import should clone the external reposito…

### PR DESCRIPTION
**Pull request by mbarkley**
…ry, not copy contents

### CODE REVIEW

**Comment by mbarkley** [comment]
Some of the code surrounding the `ImportPresenter` may seem a bit over-engineered. I had originally created a second project import screen that would work like importing examples but with history, but after discussion with @porcelli we decided not to do this now.

I left some of the generality in the code because we do plan to add a screen for importing sub-directories of a repository in the future.

**Comment by mbarkley** [comment]
@paulovmr Would you mind reviewing this?

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-backend/src/main/java/org/kie/workbench/common/screens/impl/LibraryServiceImpl.java [hunk 1]
```diff
@@ -195,6 +205,46 @@ public WorkspaceProject createProject(final OrganizationalUnit activeOrganizatio
                                          mode);
     }
 
+    @Override
+    public WorkspaceProject importProject(final OrganizationalUnit targetOU,
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-backend/src/main/java/org/kie/workbench/common/screens/impl/LibraryServiceImpl.java]: Could you please add a test for this?

Comment (mbarkley) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-backend/src/main/java/org/kie/workbench/common/screens/impl/LibraryServiceImpl.java]: Added tests in `LibraryServiceImplTest`.

**Comment by mbarkley** [comment]
@paulovmr Added tests for importing projects. Would you mind taking another look?

**Pull request merged**